.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: python
+sudo: false
```

Comment:
makes builds faster (at least, puts us in a higher throughput queue), and travis is trying to push everyone [that direction anyway](http://blog.travis-ci.com/2015-03-31-docker-default-on-the-way/).